src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -1472,13 +1472,16 @@ public final class Address implements ApiMessage {
 
     public Builder addAllUsers(List<String> users) {
       if (this.users == null) {
-        this.users = new ArrayList<>(users.size());
+        this.users = new LinkedList<>();
```

Comment:
Why the switch to `LinkedList`?

Reply:
The `add[Resources]()` and `addAll[Resources]()` methods append to the underlying list, and appending is faster for LinkedLists

Reply:
https://dzone.com/articles/arraylist-vs-linkedlist-vs

Reply:
Proto uses `ArrayList` - I might prefer to stick with what they use; they have done a fair amount of perf analysis. https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/google-api-grpc/proto-google-cloud-pubsub-v1/src/main/java/com/google/pubsub/v1/PublishRequest.java#L644